static/temp/foods.json

```diff
@@ -0,0 +1,259 @@
+            "order": 1
+          }
+        ]
+      },
+      "tips": [
+        {
+          "name": "0.2 gallons of diesel consumed",
+          "icon": "OilBarrelOutlinedIcon",
+          "description": [
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            }
+          ]
+        },
+        {
+          "name": "0.2 gallons of diesel consumed",
+          "icon": "OilBarrelOutlinedIcon",
+          "description": [
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            }
+          ]
+        }
+      ]
+    },
+    "images": ["/img1", "/img2", "/img3"],
+    "nutrition": {
+      "nutrientName": "Banana ripe, slightly ripe, raw",
+      "nutrientAmount": ["1 tablespoon", "1cup", "100g"],
+      "nutrientSource": "Source: USDA",
+      "nutrientGauges": [
+        { "name": "Protein", "percentage": "10%", "weight": "120g" },
+        { "name": "Carbs", "percentage": "20%", "weight": "120g" },
+        { "name": "Fats", "percentage": "30%", "weight": "120g" },
+        { "name": "Fiber", "percentage": "40%", "weight": "120g" }
+      ],
+      "nutrients": ["Protein", "carbohydrate", "water", "vitamin", "Fats", "Fiber", "Minerals"],
+      "allFilters": {
+        "Life Stage": [
+          {
+            "name": "Infant",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Children",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          },
+          {
+            "name": "Male",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Female",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          },
+          {
+            "name": "Pregnant",
+            "icon": "OilBarrelOutlinedIcon"
+          },
+          {
+            "name": "Location",
+            "icon": "DirectionsCarFilledOutlinedIcon"
+          }
+        ],
+        "age": ["9-13 Years", "14-18 Years", "19-30 Years", "31-50 Years", "51-70 Years"]
```

Comment:
rename to `ages`. The structure needs more depth. See below

```json
"ages": [
    {
        "start": 9,
        "startUnit": "years"
        "end": 13,
        "endUnit": "years"
        ""
    }
]
```

Create a structure that is detailed.

Reply:
done